tests/real/community.rs

```diff
@@ -70,6 +70,74 @@ async fn roundtrip() {
     assert!(drained);
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn server_state() {
+    skip_check!();
+
+    let local = "server_state";
+
+    // prepare a worker
+    let mut w = WorkerBuilder::default()
+        .register_fn(local, move |_| async move { Ok::<(), io::Error>(()) })
+        .connect(None)
+        .await
+        .unwrap();
+
+    // prepare a producing client
+    let mut client = Client::connect(None).await.unwrap();
+
+    // examine server state before pushing anything
+    let server_state = client.current_info().await.unwrap();
+    assert!(server_state.data.queues.get(local).is_none());
+    // the following two assertions are not super-helpful but
+    // there is not much info we can make meaningful assetions on anyhow
+    // (like memusage, server description string, version, etc.)
+    assert!(server_state.server.connections >= 2); // at least two clients from the current test
+    assert!(server_state.server.uptime > 0); // if IPC is happenning, this should hold :)
```

Comment:
```suggestion
    assert_ne!(server_state.server.uptime, 0); // if IPC is happenning, this should hold :)
```

Reply:
also added a couple of helpers: `assert_gt`, `assert_lt`, etc.